static/elements/chromedash-upcoming-milestone-card.js

```diff
@@ -0,0 +1,192 @@
+        <h3>
+          <span class="channel_label">Stable</span> ${this._computeDaysUntil(this.channel.stable_date)}
+          <span class="release-stable">( ${this._computeDate(this.channel.stable_date)} )</span>
+        </h3>
+      </div>
+      ` : nothing}
+    `;
+  }
+
+  _cardFeatureItemTemplate(f) {
+    return html `
+    <li data-feature-id="${f.id}">
+      <a href="/feature/${f.id}">${f.name}</a>
+      <span class="icon_row">
+        ${f.browsers.chrome.origintrial ? html`
+        <span class="tooltip" title="Origin Trial">
+          <iron-icon icon="chromestatus:extension" class="experimental" data-tooltip></iron-icon>
+        </span>
+        ` : nothing}
+        ${f.browsers.chrome.intervention ? html`
+        <span class="tooltip" title="Browser intervention">
+          <iron-icon icon="chromestatus:pan-tool" class="intervention" data-tooltip></iron-icon>
+        </span>
+        ` : nothing}
+        ${this.removedStatus.includes(f.browsers.chrome.status.text) ? html`
+        <span class="tooltip" title="Removed">
+          <iron-icon icon="chromestatus:cancel" class="remove" data-tooltip></iron-icon>
+        </span>
+        ` : nothing}
+        ${this.deprecatedStatus.includes(f.browsers.chrome.status.text) ? html`
+        <span class="tooltip" title="Deprecated">
+          <iron-icon icon="chromestatus:warning" class="deprecated" data-tooltip></iron-icon>
+        </span>
+        ` : nothing}
+        ${this.signedIn ? html`
+        <span class="tooltip"
+          title="Receive an email notification when there are updates">
+          <iron-icon
+            icon="${this.starredFeatures.includes(Number(f.id)) ?
+            'chromestatus:star' :
+            'chromestatus:star-border'}"
+            class="pushicon"
+            data-feature-id="${f.id}"
+            @click="${this.toggleStar}">
+          </iron-icon>
+        </span>
+        ` : html`
+        <span class="tooltip"
+          title="Sign in to get email notifications for updates">
+          <a href="#"  @click="${window.promptSignIn}" data-tooltip>
+            <iron-icon icon="chromestatus:star-border"
+              class="pushicon">
+            </iron-icon>
+          </a>
+        </span>
+        `}
+      </span>
+    </li>
+    
+    `;
+  }
+
+  _cardFeatureListTemplate() {
+    return html `
+      <div class="features_list">
+        <div class="features_header">${this.templateContent.featureHeader}:</div>
+
+          ${this._objKeys(this.channel.components).map((componentName) => html`
+          <h3 class="feature_components">${componentName}</h3>
+          <ul>
+            ${this.channel.components[componentName].map((f) => html`
+              ${this._cardFeatureItemTemplate(f)}
+            `)}
+          </ul>
+        `)}
+      </div>
+    `;
+  }
+
+  render() {
+    return html`
+      <section class="release ${this.showShippingType ? '' : 'no-components'}">
```

Comment:
Earlier, the true condition would return the object `nothing` . This would show up in the html as `[Object object]`. Hence, I have replaced it with empty string.